include/client.h

```diff
@@ -31,22 +31,58 @@ namespace mixer_client {
 // is completed.
 using DoneFunc = std::function<void(const ::google::protobuf::util::Status&)>;
 
+// Define transport context status
+typedef enum {
+  CTX_OK = 0,
+  CTX_NOT_EXIST,
+} CtxStatus;
+
+// Transport Context ID type.
+typedef int CtxID;
```

Comment:
Choice of nomenclature here should align with that in the API. Specifically do we intend to map this concept to 'attribute_context'

https://github.com/istio/api/blob/master/mixer/v1/attributes.proto#L83

Which is really the scope within which a client operation occurs (unless we plan to eliminate that from the API @geeknoid)